index.js

```diff
@@ -0,0 +1,111 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const models = require('./models')
+const app = express()
+
+app.get('/movies', async (request, response) => {
+  const movies = await models.movies.findAll({
+    include: [{
+      all: true
```

Comment:
I would have explicitly included Genres and Directors rather than using all. When you use all you get returned the linking table data which is redundant and just muddies things up in the resulting data.